CMakeLists.txt

```diff
@@ -13,6 +13,17 @@ include(helpers)
 
 find_package(Python3 COMPONENTS Interpreter)
 
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+# Build Options
+option(DEVELOPER_MODE "enable developer checks, treats warnings as errors" OFF)
+option(BUILD_TESTS "Build unit tests." ON)
+option(FORMAT_CPP_STYLE "format code style of C++ sources" OFF)
+option(USE_ASAN "enable AddressSanitizer" OFF)
+option(USE_UBSAN "enable UndefinedBehaviorSanitizer" OFF)
```

Comment:
ASan and UBSan can be enabled at the same time, e.g. `-fsanitize=address,undefined`, what happens if `USE_ASAN=ON` and `USE_UBSAN=ON`?

Reply:
it works fine.